skyportal/tests/frontend/test_sources.py

```diff
@@ -55,29 +56,70 @@ def test_classifications(driver, user, taxonomy_token, public_group, public_sour
         ],
     }
 
+    tax_name = str(uuid.uuid4())
+    tax_version = "test0.1"
+
     status, data = api(
         'POST',
         'taxonomy',
         data={
-            'name': str(uuid.uuid4()),
+            'name': tax_name,
             'hierarchy': simple,
             'group_ids': [public_group.id],
-            'version': "test0.1",
+            'version': tax_version,
         },
         token=taxonomy_token,
     )
     assert status == 200
 
-    driver.get(f"/become_user/{user.id}")  # TODO decorator/context manager?
+    driver.get(f"/become_user/{user.id}")
     driver.get(f"/source/{public_source.id}")
     driver.wait_for_xpath(f'//div[text()="{public_source.id}"]')
-    taxonomy_button = driver.wait_for_xpath(f'//div[@id="tax-select"]')
-    taxonomy_button.click()
+    driver.wait_for_xpath_to_be_clickable('//div[@id="tax-select"]').click()
+    driver.wait_for_xpath_to_be_clickable(
+        f'//*[text()="{tax_name} ({tax_version})"]'
+    ).click()
+    driver.wait_for_xpath("//body").click()
+    class_input = driver.wait_for_xpath('//*[@id="classification"]')
+    driver.scroll_to_element_and_click(class_input)
+    class_input.send_keys("Symmetrical", Keys.ENTER)
```

Comment:
`Symmetrical` isn't apart of the `simple` taxonomy (it is apart of `tdtax`). I'm surprised this is working.

Reply:
@profjsb 
``` python 
    simple = {
        'class': 'Cepheid',
        'tags': ['giant/supergiant', 'instability strip', 'standard candle'],
        'other names': ['Cep', 'CEP'],
        'subclasses': [
            {'class': 'Anomolous', 'other names': ['Anomolous Cepheid', 'BLBOO']},
            {
                'class': 'Mult-mode',
                'other names': ['Double-mode Cepheid', 'Multi-mode Cepheid', 'CEP(B)'],
            },
            {
                'class': 'Classical',
                'tags': [],
                'other names': [
                    'Population I Cepheid',
                    'Type I Cepheid',
                    'DCEP',
                    'Delta Cepheid',
                    'Classical Cepheid',
                ],
                'subclasses': [
                    {
                        'class': 'Symmetrical',
                        'other names': ['DCEPS', 'Delta Cep-type Symmetrical'],
                    }
                ],
            },
        ],
    }
```
I believe you wrote this.